iogt/templates/base.html

```diff
@@ -27,6 +27,22 @@
     <meta name="description" content=""/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
     <meta name="theme-color" content="#493174">
+    <meta property="og:url" content="{{ page.url }}">
+    <meta property="og:site_name" content="{{ page.get_site.site_name }}">
+    <meta property="og:title" content="{{ page.title }}">
+    <meta property="og:description" content="Internet of Good Things.">
```

Comment:
Description of the article/page or the site? Can we do better than 'Internet of Good Things'?

Reply:
@istride, there may be some articles or pages that do not have descriptions, it's not possible to create dynamic content for them. Do you have any suggestions for a term that could be used instead of 'Internet of Good Things'?